src/codeu/chat/client/simplegui2/ChatMenu.java

```diff
@@ -0,0 +1,200 @@
+        mainWindow.setCenter(centralTextBox());
+
+    }
+
+    private HBox topMenuBar(){
+
+        //create menu at top
+        HBox topbar= new HBox(10);
+        topbar.setMaxHeight(100);
+        topbar.setPadding(new Insets(15, 12, 15, 12));
+        topbar.setStyle("-fx-background-color: #336699;");
+
+        //add buttons for different options
+        final Button signOut = new Button("Sign Out");
+        signOut.setPrefSize(100, 20);
+
+        final Button addUser = new Button("Add User");
+        addUser.setPrefSize(100, 20);
+
+        final Button newConvo = new Button("New Conversation");
+        newConvo.setPrefSize(100, 20);
+
+        signOut.setOnAction(new EventHandler<ActionEvent>() {
+            @Override
+            public void handle(ActionEvent actionEvent) {
+
+                if (clientContext.user.hasCurrent()) {
+
+                    boolean successfulSignOut= clientContext.user.signOutUser();
+
+                } else {
+                    AlertBox alertNoUser = new AlertBox("You are not signed in.", "You cannot sign out without being logged in.");
+                    alertNoUser.display();
+                }
+            }
+        });
+
+        addUser.setOnAction(new EventHandler<ActionEvent>() {
+            @Override
+            public void handle(ActionEvent actionEvent) {
+                NewUser user1=new NewUser(clientContext);
+            }
+        });
+
+        newConvo.setOnAction(new EventHandler<ActionEvent>() {
+            @Override
+            public void handle(ActionEvent actionEvent) {
+
+                if (clientContext.user.hasCurrent()) {
+
+                    NewConversation addConversationBox = new NewConversation(clientContext, "Add Conversation", "Enter new conversation name:" );
+                    addConversationBox.enterNewConversation();
+                    String s = addConversationBox.conversationInput();
+
+                    if (s != null && s.length() > 0) {
+                        clientContext.conversation.startConversation(s, clientContext.user.getCurrent().id);
+                        //ConversationPanel.this.getAllConversations(listModel);
+                    }
+                } else {
+                    AlertBox alertNoUser = new AlertBox("You are not signed in.", "Please sign in before joining a conversation.");
+                    alertNoUser.display();
+                }
+            }
+        });
+
+
+        //add buttons to menu
+        topbar.getChildren().addAll(signOut, addUser, newConvo);
+
+        return topbar;
+    }
+
+    private static VBox sideBar(){
+
+        //create sidebar
+        VBox sidebar= new VBox(5);
+
+        sidebar.setPadding(new Insets(3));
+        sidebar.setAlignment(Pos.TOP_LEFT);
+        sidebar.setMinWidth(200);
+
+        //Label for box
+        Label sideLabel=new Label("Conversations");
+
+        //border for box
+        sidebar.setStyle("-fx-padding: 10;" +
```

Comment:
Consider:  Writing a class to encapsulate style key:value pair definitions.  Using strings like this can be very error prone.